src/EnergyPlus/ChillerElectricASHRAE205.cc

```diff
@@ -1443,13 +1429,22 @@ void ASHRAE205ChillerSpecs::calculate(EnergyPlusData &state, Real64 &MyLoad, boo
     } else if (this->ChillerPartLoadRatio < 1.0) // Modulating
     {
         // Use performance map to find the fractional sequence number (which most closely matches our part load)
-        auto f = std::bind(&ASHRAE205ChillerSpecs::findCapacityResidual, this, std::placeholders::_1, std::placeholders::_2, std::placeholders::_3);
         Real64 constexpr accuracy{0.0001};
         int constexpr maxIter{500};
         int solFla{0};
-        std::array<Real64, 4> par{{MyLoad, RunFlag ? 1.0 : 0.0, 1.0}}; // Initialize iteration parameters for RegulaFalsi function
+        auto f = [&MyLoad, this](Real64 partLoadSeqNum) {
```

Comment:
In cases where we are accessing a residual member function, we can simply capture the `this` element and call member functions as needed.  No need to use `std::bind` to forward the call.  Also in this case note that `state` isn't needed, so no need to capture it.  Keeps things minimal.